### DISCUSSION TITLE
Can't connect a second TTN instance

### DISCUSSION THREAD

**Discussion by bjvt** [emqx/MQTTX#1528] [Q&A]
I can connect to my TheThingsNetwork with MQTTX with Application 1 communicating well.  However, Application 2 with its username/password does not connect?  I can't find any my log file (Current is empty).  Running MQTTX on SNAP Ubuntu.  I know the user/pass is correct because NodeRed, MQTT Explorer, and mosquito_sub work fine.

**Comment by ysfscream** [comment]
Hello,

Thank you for submitting this issue. I would like to understand more about your problem, but I need clarification on the details. To help you better, I would need more specific information, especially regarding the steps you took to connect to the second TTN instance. Please provide the detailed steps for setting up and attempting to join the second application in MQTTX. This information will help us to reproduce and diagnose the issue accurately.

Looking forward to your reply, thank you!
